packages/flutter/lib/src/material/divider.dart

```diff
@@ -117,9 +117,20 @@ class Divider extends StatelessWidget {
   /// ```
   /// {@end-tool}
   static BorderSide createBorderSide(BuildContext context, { Color color, double width }) {
+    final Color themeColor = context != null ? (DividerTheme.of(context).color ?? Theme.of(context).dividerColor) : null;
+    final double themeWidth = context != null ? DividerTheme.of(context).thickness : null;
+    final double resolvedWidth = width ?? themeWidth ?? 0.0;
+
```

Comment:
consider adding 

`final Color effectiveColor = color ?? themeColor'`

Then below `color == null && themeColor == null` could be `effectiveColor == null`

Reply:
Done